libs/sql-schema-describer/src/sqlite.rs

```diff
@@ -363,7 +363,9 @@ impl SqlSchemaDescriber {
         let filtered_rows = result_set
             .into_iter()
             // Exclude primary keys, they are inferred separately.
-            .filter(|row| row.get("origin").and_then(|origin| origin.as_str()).unwrap() != "pk");
+            .filter(|row| row.get("origin").and_then(|origin| origin.as_str()).unwrap() != "pk")
+            // Exclude partial indices
+            .filter(|row| row.get("partial").and_then(|partial| partial.as_bool()).unwrap() != true);
```

Comment:
mmh does this mean we can't introspect what the predicate for filtering which rows are in the index on sqlite? Good to know

Reply:
At least not from there. I guess there must be a way to get it from the ddl, but the list_indexes Pragma does not have that info.